test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/MiddlewareFilterConfigurationProviderTest.cs

```diff
@@ -174,5 +187,9 @@ private void Configure(IApplicationBuilder appBuilder)
 
             }
         }
+
+        private abstract class AbstractType_NoConstructor
```

Comment:
`AbstractType` should suffice. The presence or absence of ctor shouldn't matter.